helm/micro-integrator/templates/_helpers.tpl

```diff
@@ -55,3 +55,12 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
+
+{{/*
+Micro Integrator Managment Host Name
+*/}}
+
+{{- define "micro-integrator.ingress.mgt.hostname" -}}
+{{- $mgtDomain := .Values.wso2.ingress.mgt.host | default .Release.Name -}}
```

Comment:
@ThilinaManamgoda shall we set some default name such as, `mi.wso2.com` rather than using the release name since this usually does not amount to be a proper DNS hostname?

Reply:
And also, @ThilinaManamgoda shall we move all default values to the user input `values.yaml` file?